### PR TITLE
Add vagarwal to triage

### PR DESCRIPTION
**Pull request by vipulagarwal**
#### Summary
Requesting membership with `triage` permissions as discussed with @bobcallaway. Thank you!

### CODE REVIEW

File: github-sync/github-data/sigstore/users.yaml [hunk 1]
```diff
@@ -544,3 +544,8 @@ users:
     teams:
       - name: gitsign-codeowners
         role: member
+  - username: vagarwal
+    role: member
+    teams:
+      - name: triage
```
Comment (haydentherapper) [inline github-sync/github-data/sigstore/users.yaml:550]: Can we add you to a specific repository, I assume helm-charts? This is triage across all repositories.

Comment (vipulagarwal) [reply github-sync/github-data/sigstore/users.yaml:550]: Yes, just need helm-charts at the moment. Shall I use `teams.name: helm`?

Comment (haydentherapper) [reply github-sync/github-data/sigstore/users.yaml:550]: Add yourself to this file but make teams `teams: []`, and then add yourself as a collaborator in the helm-charts section like https://github.com/sigstore/community/blob/main/github-sync/github-data/sigstore/repositories.yaml#L714-L715

Comment (vipulagarwal) [reply github-sync/github-data/sigstore/users.yaml]: I have made the changes, thank you

**Review by haydentherapper — APPROVED** [review]

**Comment by vipulagarwal** [comment]
@haydentherapper @bobcallaway can we merge this please? thanks

**Comment by bobcallaway** [comment]
> @haydentherapper @bobcallaway can we merge this please? thanks

@sigstore/core-team please weigh in

**Review by SantiagoTorres — APPROVED** [review]
Did a quick review of @vipulagarwal 's contributions in the last ~12 months. LGTM

**Review by bobcallaway — APPROVED** [review]

**Review by lukehinds — APPROVED** [review]

**Pull request merged**